examples/service-per-ns/sync.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
+import json
+import copy
+import re
+import sys
+
+def is_job_finished(job):
```

Comment:
I'm assuming you planned to clean up the indexedjob remnants later; just leaving this comment to remind us.

Reply:
Yes! I'll make sure to do that before I forget!